exercises/practice/gigasecond/.docs/hints.md

```diff
@@ -0,0 +1,3 @@
+# Hints
+
+* You can import Foundation for time and date functions
```

Comment:
```suggestion
- You can import `Foundation` for time and date functions
```

Reply:
That came to mind while falling asleep last night actually!